backend/controllers/homepage/homepage.ts

```diff
@@ -277,6 +279,34 @@ const getTasks = ({ activeCaseLoadId, locations, staffId, whereaboutsConfig, key
         createAndVaryALicence.enabled_prisons.split(',').includes(activeCaseLoadId) &&
         userHasRoles(['LICENCE_CA', 'LICENCE_DM', 'LICENCE_RO', 'LICENCE_ACO', 'LICENCE_ADMIN']),
     },
+    {
+      id: 'activities',
+      heading: 'Allocate people to activities',
+      description: 'Set up and edit activities. Allocate people, remove them, and edit allocations.',
+      href: activities.url,
+      enabled: () => activities.url && activities.enabled_prisons.split(',').includes(activeCaseLoadId),
+    },
+    {
+      id: 'appointments',
+      heading: 'Schedule and edit appointments',
+      description: 'Create one-to-one and group appointments. Edit existing appointments and print movement slips.',
+      href: appointments.url,
+      enabled: () => appointments.url && appointments.enabled_prisons.split(',').includes(activeCaseLoadId),
+    },
+    {
+      id: 'view-covid-units',
+      heading: 'View COVID units',
+      description: 'View who is in each COVID unit in your establishment.',
+      href: '/current-covid-units',
+      enabled: () => userHasRoles(['PRISON']) && activities.enabled_prisons.split(',').includes(activeCaseLoadId),
+    },
```

Comment:
COVID units - ok to leave but there has been mention that it could be dropped. Would need someone to definitely say yes or no.